simulation/src/LeiosProtocol/Short.hs

```diff
@@ -680,6 +686,26 @@ inputBlocksToEndorse cfg@LeiosConfig{pipeline = _ :: SingPipeline p} current buf
         , receivedBy
         }
 
+-- | Invokes 'inputBlocksToEndorse1' as many times as 'lateIbInclusion'
```

Comment:
TODO this isn't going to work, since the IBs will have been pruned.

I'll need to relax the pruning logic https://github.com/input-output-hk/ouroboros-leios/blob/b5dd5b4440b01d79960bf6de6573dcecfbce8d3a/simulation/src/LeiosProtocol/Short/Node.hs#L537